src/components/TripPlanner/TripPlanner.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import './TripPlanner.scss'
+
+export function TripPlanner() {
+    return (
+        
+        <section className ="tripplanner-container">
+            <div className="tripplanner-header">Please enter your destination for whale spotting!</div>
+            <div className="tripplanner-search">
+            SEARCH
+            </div>
+        <div className="tripplanner-map"> MAP</div>
+        <div className="tripplanner-table">TABLE</div>
```

Comment:
Please can you change the formatting and indentation to make this easier to read, eg.
```suggestion
            <div className="tripplanner-header">
                Please enter your destination for whale spotting!
            </div>
            <div className="tripplanner-search">
                SEARCH
            </div>
            <div className="tripplanner-map">
                MAP
            </div>
            <div className="tripplanner-table">
                TABLE
            </div>
```